lib/ES/Book.pm

```diff
@@ -408,7 +408,7 @@ sub _update_title_and_version_drop_downs {
 
         # If a book uses a custom index page, it may not include the TOC. The
         # substitution below will fail, so we abort early in this case.
-        next unless ($_ == 'index.html' && ($html =~ /ul class="toc"/));
+        next unless ($_ eq 'index.html' && ($html =~ /ul class="toc"/));
```

Comment:
This was complaining. I think this is the right fix.

Reply:
I tried this before but had to remove it in 8afdcf6efbf52815a565d1142a267f2eae2d95fc. I don't remember the specifics but it's been on my "don't touch this unless you have time to figure out what's going on" list.

Reply:
I'll revert. cool.